<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListeTaches</class>
 <widget class="QDialog" name="ListeTaches">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>426</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OGC (Outil de Gestion des Contacts)</string>
  </property>
  <widget class="QLabel" name="titre">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Liste des tâches</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonRetour">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateDebut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Afficher entre le</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateDebut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximumDate">
    <date>
     <year>2026</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateFin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximumDate">
    <date>
     <year>2026</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateFin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>et le</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SupprimerTache">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Supprimer tâche</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableListeTaches">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>711</width>
     <height>261</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>190</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>190</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>35</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>35</number>
   </attribute>
   <column>
    <property name="text">
     <string>IdTache</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date de réalisation</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Contenu de la tâche</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelListeTaches">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Liste des tâches associées au contact :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nomContact">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>48</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
